elements/pfe-accordion/demo/index.html

```diff
@@ -18,81 +20,181 @@
     <script src="/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.min.js"></script>
     <script>require([
-      '../pfe-accordion.umd.js'
+
+      '../pfe-accordion.umd.js',
+      '../../pfe-band/pfe-band.umd.js'
     ])</script>
+
+    <style>
+      body {
+        font-family: Overpass;
+        padding: 16px 32px 32px;
+      }
+      section {
+        padding: 2rem;
+      }
+      .dark-background {
+        background-color: #191e84;
+      }
+      .white-background {
+        background-color: #fff;
+      }
+      .med-background {
+        background-color: #9394ba;
+      }
+      .dark-gray-background {
+        background-color: #222591;
+      }
+      .light-gray-background {
+        background-color: #f8f8f8;
+      }
+
+    </style>
   </head>
   <body unresolved>
-    <h1>&lt;pfe-accordion&gt;</h1>
-    <section>
-      <pfe-accordion color="lightest">
+
+    <section class="light-gray-background">
+      <h2>&lt;pfe-accordion&gt;</h2><br/>
+      <pfe-accordion>
         <pfe-accordion-header>
-          <h2>Why do wizards need money if they could just create it?</h2>
+          <h3>Why do wizards need money if they could just create it?</h3>
         </pfe-accordion-header>
         <pfe-accordion-panel>
           <p>There is legislation that decides what you can conjure and what you can not. Because things that you conjure out of thin air will not last, it is illegal in the wizarding world.</p>
         </pfe-accordion-panel>
         <pfe-accordion-header>
-          <h2>Why doesn't Harry have a portrait of his parents?</h2>
+          <h3>Why doesn't Harry have a portrait of his parents?</h3>
         </pfe-accordion-header>
         <pfe-accordion-panel>
-          <p>The characters in the portraits are not actually ghosts. They mainly are there just to repeat common phrases or serve as a general representation of the individual they depict. A portrait of his parents would not be of much help to Harry.</p>
+          <p><a href="#">The characters in the portraits</a> are not actually ghosts. They mainly are there just to repeat common phrases or serve as a general represe<a href="foobarbaz.com">ntation of the individual</a> they depict. A portrait of his parents would not be of much help to Harry.</p>
```

Comment:
Do we need this second link in the middle of the text?  It's a little funky splitting up the word and encapsulating multiple words here. :)

Reply:
@castastrophe Weird. Not sure how that happened but I fixed all of these.